src/components/ArticleListItem.jsx

```diff
@@ -0,0 +1,15 @@
+import { Link } from "react-router-dom";
+import { BsFillArrowRightCircleFill } from "react-icons/bs";
+
+const ArticleListItem = ({ article }) => {
+  return (
+    <div className="articleItem__container">
+      <h4>{article.title}</h4>
```

Comment:
```jsx
    <div className="articleItem__container">
      <h4>{article.title}</h4>
```
Semantically these tags should be an `li` and a `h2` if they're subheadings in a list.